packages/realm-app-importer/tsconfig.build.json

```diff
@@ -1,9 +1,9 @@
 {
 	"extends": "./tsconfig.json",
 	"compilerOptions": {
-		"target": "ES5",
+		"target": "es2018",
```

Comment:
Can we go straight for `es2020`?

Reply:
Most likely, but I didn't have to for now.